src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Network/DnsZone/New-ArmPrivateDnsZone.ps1

```diff
@@ -0,0 +1,56 @@
+function New-ArmPrivateDnsZone {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("PDNSZ")]
```

Comment:
This name should be more explicit

Reply:
This is following the previous convention in `New-ArmDnsZone.ps1`